Makefile

```diff
@@ -308,7 +322,7 @@ endef
 .PHONY: tags
 tags:
 	@echo "## xNVMe: make tags"
-	@$(CTAGS) * --languages=C -h=".c.h" -R --exclude=builddir \
+	@$(CTAGS) * --languages=C -h=".c.h" -R --exclude=$(BUILD_DIR) \
```

Comment:
I really do prefer that unrelated changes, such as these fixing the hardcoded builddir, would go into a separate commit.

Reply:
Moved to separate commit